code/game/objects/items/robot/robot_items.dm

```diff
@@ -546,9 +546,11 @@
 		to_chat(user, "<span class='boldwarning'>[src] is still recycling its projectors!</span>")
 		return
 	cycle_delay = world.time + PKBORG_DAMPEN_CYCLE_DELAY
-	active = !active
-	if(active)
-		activate_field(user)
+	if(!active)
+		if(!user.has_buckled_mobs())
+			activate_field()
+		else
+			to_chat(user, "<span class='warning'>[src] refuses to activate!</span>")

```

Comment:
I feel like this should give a reason on why it refuses to activate.

Reply:
kk